embedded.go

```diff
@@ -0,0 +1,11 @@
+package embeddedfiles
+
+import _ "embed"
```

Comment:
is the underscore needed?

Reply:
yes

Reply:
Ok I understand now. The example I linked actually called the library, which is why it didn't need the underscore